web/src/components/Registration.js

```diff
@@ -0,0 +1,267 @@
+                            />
+                          </Card.Body>
+                        </Accordion.Collapse>
+                      </Card>
+                      <Card>
+                        <Accordion.Toggle as={Card.Header} eventKey="administrative">
+                          Administrative Contact
+                        </Accordion.Toggle>
+                        <Accordion.Collapse eventKey="administrative">
+                          <Card.Body>
+                            <p>
+                              Administrative or executive contact for your organization to field
+                              high-level requests or queries. (Optional).
+                            </p>
+                            <Contact
+                              contact={this.state.formData.contacts.administrative}
+                              onChange={this.createChangeHandler("administrative", "contacts")}
+                            />
+                          </Card.Body>
+                        </Accordion.Collapse>
+                      </Card>
+                      <Card>
+                        <Accordion.Toggle as={Card.Header} eventKey="billing">
+                          Billing Contact
+                        </Accordion.Toggle>
+                        <Accordion.Collapse eventKey="billing">
+                          <Card.Body>
+                            <p>
+                              Billing contact for your organization to handle account and invoice
+                              requests or queries relating to the operation of the TRISA network. (Optional).
+                            </p>
+                            <Contact
+                              contact={this.state.formData.contacts.billing}
+                              onChange={this.createChangeHandler("billing", "contacts")}
+                            />
+                          </Card.Body>
+                        </Accordion.Collapse>
+                      </Card>
+                    </Accordion>
+
+                    <Form.Group>
+                      <Button variant="secondary" onClick={(e) => this.setState({tabKey:"trisa-implementation"})}>Back</Button>{' '}
+                      <Button variant="primary" onClick={(e) => this.setState({tabKey:"trixo"})}>Next</Button>
+                    </Form.Group>
+                  </fieldset>
+                </Tab.Pane>
+                <Tab.Pane eventKey="trixo">
+                  <fieldset>
+                    <legend>TRIXO Questionnaire</legend>
+                    <p>
+                      This questionnaire is designed to help the TRISA working group and other
+                      TRISA members understand the regulatory regime of your organization to ensure
+                      that required compliance information exchanges are conducted correctly and
+                      safely. All verified TRISA members will have access to this information.
```

Comment:
Recommend breaking the text up into shorter sentences to support translation:

This questionnaire is designed to help the TRISA working group and TRISA members understand the regulatory regime of your organization. The information you provide will help ensure that required compliance information exchanges are conducted correctly and safely. All verified TRISA members will have access to this information.

Reply:
Great point!